src/cli/repo-checkout.js

```diff
@@ -152,7 +165,18 @@ export default async function main(config, argv) {
   // Extraction of archives should *not* be done in parallel because of race
   // conditions with writing to the same directories.
   for (let archive of archivesToExtract) {
-    if (!archive.archivePath) continue;
+    if (!archive) continue;
```

Comment:
It was a one liner to ignore null entries in archivesToExtract which happens when there was no archive downloaded (because it was done so previously).